src/ProjectTemplates/test/Helpers/ProcessEx.cs

```diff
@@ -104,14 +104,14 @@ public static ProcessEx Run(ITestOutputHelper output, string workingDirectory, s
             return new ProcessEx(output, proc);
         }
 
-        public static async Task<ProcessEx> RunViaShellAsync(ITestOutputHelper output, string workingDirectory, string commandAndArgs)
+        public static ProcessEx RunViaShell(ITestOutputHelper output, string workingDirectory, string commandAndArgs)
```

Comment:
Why no longer async?

Reply:
`await result.Exited` was the last async thing in this function, without it we get that annoying warning/error about an async with nothing awaited. Unfortunately I wasn't able to find/manufacture an overload to wait for result to complete that was both async and did a timeout.